docker-build-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Builds the APK inside the Docker container
+
+set -o errexit
+
+cd /repo
+
+npm cache verify
+npm install
+npx gulp
+npx cordova prepare
+
+export ANDROID_HOME=/usr/lib/android-sdk
+npx cordova build android --release
+
+mkdir -p /dist/apk
```

Comment:
Duplicate, this directory is already being created in the Dockerfile.

Reply:
Right, it shouldn't be in the Dockerfile (where the outside path isn't mounted). I've removed it there.